embulk-output-mysql/src/main/java/org/embulk/output/mysql/MySQLOutputConnection.java

```diff
@@ -101,6 +101,29 @@ protected String buildCollectMergeSql(List<TableIdentifier> fromTables, JdbcSche
         return sb.toString();
     }
 
+    @Override
+    protected String buildSwapTablesSql(TableIdentifier fromTable, TableIdentifier toTable)
+    {
+        TableIdentifier tmpTable = new TableIdentifier(fromTable.getDatabase(), fromTable.getSchemaName(), fromTable.getTableName() + "_embulk_swap_tmp");
+
```

Comment:
Does this table name use unique temporary table. Is it safe if `target_table_embulk_swap_tmp` already exists?

Reply:
>  Is it safe if target_table_embulk_swap_tmp already exists?

No. so it should be fixed as a unique identifier.